dev/Expander/Expander.xaml

```diff
@@ -23,14 +23,13 @@
         <Setter Property="Background" Value="{ThemeResource ExpanderBackground}" />
         <contract7Present:Setter Property="BackgroundSizing" Value="InnerBorderEdge" />
         <Setter Property="MinWidth" Value="{ThemeResource FlyoutThemeMinWidth}" />
-        <Setter Property="MaxWidth" Value="{ThemeResource FlyoutThemeMaxWidth}" />
         <Setter Property="MinHeight" Value="{StaticResource ExpanderMinHeight}" />
         <Setter Property="BorderThickness" Value="{ThemeResource ExpanderBorderThickness}" />
         <Setter Property="BorderBrush" Value="{ThemeResource ExpanderBorderBrush}" />
         <Setter Property="Padding" Value="{StaticResource ExpanderHeaderPadding}" />
```

Comment:
@tashatitova Should this really be using the "HeaderPadding"? 

We have two paddings we need to worry about and you already have resources defined for these `ExpanderHeaderPadding` and `ExpanderContentPadding`. Now you are setting the padding of the control itself... to be the header padding. This is opposite of what I would expect. Padding is for the Contnet, not the header.

This means ContentPresenter needs to bind to the control's padding, the control's padding needs to be set to ExpanderContentPadding by default and the header toggle button needs to set it's padding directly. Basically just switch everything opposite from how it is.

This is for the control itself. We need both a content padding and a heading padding separately. Instead the HeaderPadding should be set in the ToggleButton itself. Padding should be whatever the

Reply:
Edit: We have the same issue with Background. The Header is what is bound to the control's Background and Padding properties. However, the expectation (most strongly for padding) is that this would apply to the content. Background itself may need to be thought about differently as mentioned elsewhere (in WPF it applies behind the header and content).

Reply:
well the expander in collapsed state is what the control looks like in default state, so I'd expect that its background and padding would be set on that header element. Content can be completely custom, with any sort of visuals and padding and what not.

Reply:
> so why is default MaxWidth gone from overall control style now? I don't believe that's what we want.

The simple answer is because with a MaxWidth set any developer settings HorizontalAlignment=Stretch will not get the functionality they expect. All other content controls do NOT have a MaxWidth specified like this. There is no limit to the width of Button, Border, StackPanel, ComboBox, the list goes on. I have to think adding one here is incorrect.

Reply:
> well the expander in collapsed state is what the control looks like in default state, so I'd expect that its background and padding would be set on that header element.

I had the same initial thought. Commented on that below -- Padding is the issue and then we need consistency.

Reply:
We can't compare this control to border or stack panel. Dialogs, flyouts, tooltips have a max width and we need one here as well. Developer can override it, same as for dialog, flyout, etc

Reply:
ContextMenu is another control with maswidth. There is no one fits all.

Reply:
Header text is expected to wrap.

Reply:
I do agree there is never a one size fits all. However, this isn't in the same category as the other controls you mentioned.  Generic content controls don't set a MaxWidth like this -- especially for the header. There seems to be well established convention about this.

Note in my tests I noticed something else. WPF always sizes the Expander to take whatever space is available -- it doesn't size to content.

Reply:
> MaxWidth should not be set so that the control sizes to content and the layout properties work as expected. 

This is already done so no additional changes are necessary.

> the options are to either map both to the **control**, or pick one or the other to template bind to and expose the other theme resources so that they can be customized.

Mapping both to the control doesn't really make sense here -- although WPF did that for some such as Background. More on this below.

Reply:
Relevant control properties now bind to the content area.